app/controllers/users_controller.rb

```diff
@@ -1,28 +1,72 @@
 class UsersController < ApplicationController
   include Pagination
-  before_action :set_user, only: [:show, :update, :destroy]
+  include Rack::Utils
+  before_action :set_user, only: [:show, :update]
 
-  # GET /users/1
+  # GET /users/:id
   def show
     render json: UserSerializer.new(@user).serialized_json
   end
 
-  # PATCH/PUT /users/1
+  # PATCH/PUT /users/:id
   def update
     unless @user.id == auth_user.id
-      return render json: ErrorSerializer.new('You can update only your profile', Rack::Utils.status_code(:forbidden)).serialized_json, status: :forbidden
+      return render json: ErrorSerializer.new('Puoi aggiornare solo il tuo profilo', status_code(:forbidden)).serialized_json, status: :forbidden
     end
 
     if @user.update(user_params)
       render json: UserSerializer.new(@user).serialized_json
     else
-			render json: ErrorSerializer.new(@user.errors, Rack::Utils.status_code(:unprocessable_entity)).serialized_json, status: :unprocessable_entity
+			render json: ErrorSerializer.new(@user.errors, status_code(:unprocessable_entity)).serialized_json, status: :unprocessable_entity
     end
   end
 
-  # DELETE /users/1
-  def destroy
-    @user.destroy
+  # POST /projects/:project_id/members
+  def members_create
+    project = Project.includes(:admins).find(params[:project_id])
+    collaborators = User.find(params[:users][:ids])
+    current_user = auth_user
+
+    @serializer_options[:params] = {project_id: params[:project_id]}
+
+    unless project.admins.detect {|admin| admin.id == current_user.id }
+      return render json: ErrorSerializer.new('Non puoi aggiungere membri se non sei l\'admin del progetto', status_code(:forbidden)).serialized_json, status: :forbidden
+    end
+
+    if project.collaborators << collaborators
+      render json: UserSerializer.new(project.members.includes(:projects_users), @serializer_options).serialized_json
+    else
+      render json: ErrorSerializer.new(project.errors, status_code(:unprocessable_entity)).serialized_json, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /projects/:project_id/members/:id
```

Comment:
Nella delete passiamo gli ids nel payload, come per la creazione

Reply:
Ciò prevede lato frontend che uno selezioni gli utenti da cancellare e che poi confermi la cancellazione; trovo più user friendly che uno possa cancellare gli utenti con un solo click

Reply:
Lato frontend possiamo anche cancellarli uno per volta, il backend funziona in entrambi i casi, esattamente come avviene nella create

Reply:
Okz, cambio